app/src/androidTest/java/org/simple/clinic/bloodsugar/BloodSugarRepositoryAndroidTest.kt

```diff
@@ -48,22 +48,23 @@ class BloodSugarRepositoryAndroidTest {
 
```

Comment:
Reword the commit message?

Ex: "Return `BloodSugarMeasurement` after creating it"